components/primary_button/README.md

```diff
@@ -70,7 +70,7 @@ As read using ChromeVox
     text-decoration: none;
     display: block;
     font-size: 18px;
-    font-family: ‘Noto Sans’, Verdana, Arial, sans-serif;
+    font-family: BCSans, ‘Noto Sans’, Verdana, Arial, sans-serif;
```

Comment:
I think we need quotes around BCSans

Reply:
You are right. Sorry I missed that

Reply:
My mistake - I've wrapped BCSans in quotes, as it is a non-generic font-family/required by CSS

Reply:
@davidcusack  You had it right in the issue - I omitted the quotes, in error